modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/DatePicker/DatePicker.es.js

```diff
@@ -12,239 +12,145 @@
-	name: 'dateField',
-	spritemap,
-};
-
-const DatePickerWithProvider = (props) => (
-	<PageProvider value={{editingLanguageId: 'en_US'}}>
-		<DatePicker {...props} />
-	</PageProvider>
-);
-
 describe('DatePicker', () => {
-	// eslint-disable-next-line no-console
-	const originalWarn = console.warn;
-
-	beforeAll(() => {
-		// eslint-disable-next-line no-console
-		console.warn = (...args) => {
-			if (
-				/DataProvider: Trying/.test(args[0]) ||
-				/Deprecation warning: value provided is not in a recognized RFC2822 or ISO format/.test(
-					args[0]
-				)
-			) {
-				return;
-			}
-			originalWarn.call(console, ...args);
-		};
-	});
-
-	afterAll(() => {
-		// eslint-disable-next-line no-console
-		console.warn = originalWarn;
-	});
-
-	afterEach(cleanup);
-
 	beforeEach(() => {
-		jest.useFakeTimers();
 		fetch.mockResponseOnce(JSON.stringify({}));
 	});
 
+	afterEach(cleanup);
+
 	it('has a helptext', () => {
 		const {container} = render(
-			<DatePickerWithProvider
-				{...defaultDatePickerConfig}
-				tip="Type something"
-				value="06/02/2020"
-			/>
+			<DatePicker tip="Type something" value="06/02/2020" />
 		);
 
-		act(() => {
-			jest.runAllTimers();
-		});
-
 		expect(container.querySelector('.form-text')).toBeTruthy();
 	});
 
 	it('has a label', () => {
-		const {container} = render(
-			<DatePickerWithProvider
-				{...defaultDatePickerConfig}
-				label="label"
-			/>
-		);
-
-		act(() => {
-			jest.runAllTimers();
-		});
+		const {container} = render(<DatePicker label="label" />);
 
 		expect(container.querySelector('.ddm-label')).toBeTruthy();
 	});
 
 	it('has a predefinedValue', () => {
 		const {container} = render(
-			<DatePickerWithProvider
-				{...defaultDatePickerConfig}
-				predefinedValue="06/02/2020"
+			<DatePicker
+				label="date"
+				locale="en_US"
+				localizedValue={{}}
+				name="dateField"
+				predefinedValue="2020-06-02"
 			/>
 		);
 
-		act(() => {
-			jest.runAllTimers();
-		});
+		const inputDatePicker = container
+			.getElementsByClassName('form-control')
+			.item(0);
 
-		expect(container).toMatchSnapshot();
+		expect(inputDatePicker).toHaveValue('06/02/2020');
 	});
 
-	it('expands the datepicker when clicking the calendar icon', async () => {
-		const {container} = render(
-			<DatePickerWithProvider {...defaultDatePickerConfig} />
-		);
+	it('expands the datepicker when clicking the calendar icon', () => {
+		const {container} = render(<DatePicker />);
 
-		userEvent.click(
-			container.querySelector('.date-picker-dropdown-toggle')
+		const btnDropdown = container.querySelector(
+			'.date-picker-dropdown-toggle'
 		);
 
-		act(() => {
-			jest.runAllTimers();
-		});
+		userEvent.click(btnDropdown);
 
-		await wait(() =>
-			expect(
-				document.body.querySelector('.date-picker-dropdown-menu.show')
-			).toBeTruthy()
+		const dropdownDatePicker = document.body.querySelector(
+			'.date-picker-dropdown-menu.show'
 		);
+
+		expect(dropdownDatePicker).toBeTruthy();
```

Comment:
Prefe use [jest-dom assertions](https://github.com/testing-library/jest-dom) for a proper error messages whenever the test fail:
```suggestion
		expect(dropdownDatePicker).tobeinthedocument();
```